.travis.yml

```diff
@@ -1,8 +1,8 @@
 sudo: false
 language: go
 go:
-  - 1.10.0
-  - 1.6.2
+  - 1.11.8
+  - 1.12.3
```

Comment:
We can simplify this to always use the latest patch versions and not having to update the travis config when new ones come out:

```
- "1.12.x"
- "1.11.x"
```